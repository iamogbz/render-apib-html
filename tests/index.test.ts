// import * as mockEvent from "./mocks/helloWorldEvent.json";
// import { handler } from "../src";

describe("entry", () => {
    it.skip("runs a test", async () => {
        // expect(await handler(mockEvent)).toMatchSnapshot();
    });
});
