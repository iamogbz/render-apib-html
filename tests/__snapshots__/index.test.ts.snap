// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`entry runs a test 1`] = `
Object {
  "body": "Hello world!",
  "status": "200",
}
`;
